kubernetes_platform/proto/kubernetes_executor_config.proto

```diff
@@ -38,6 +38,7 @@ message KubernetesExecutorConfig {
     repeated NodeAffinityTerm node_affinity = 14;
     repeated PodAffinityTerm pod_affinity = 15;
     EnabledSharedMemory enabled_shared_memory = 16;
+    repeated EmptyDirMount empty_dir_mount = 17;

```

Comment:
nit: empty_dir_mounts

Reply:
interesting. I don't see plurals anywhere else -- for example, line 26:
```
repeated PvcMount pvc_mount = 3;
```

that said, this is my very first time doing anything with protobuf, so if repeated fields should be pluralized, I will pluralize it.

Reply:
ref: https://cloud.google.com/apis/design/naming_convention#repeated_field_names

> Repeated fields in APIs must use proper plural forms. This matches the convention of existing Google APIs, and the common expectation of external developers.